### PR TITLE
Loc fixed.

### PR DESCRIPTION
**Pull request by joelAVS**
Kunosato's loc fixes for the Germany and Hungary event chain.

### CODE REVIEW

File: TGC/events/Germany VS Hungary Events.txt [hunk 1]
```diff
@@ -1795,7 +1795,7 @@
 		}
 	}
 
-## Siebenb�rgen conference - Embarresment ##
+## Siebenb?rgen conference - Embarresment ##
```
Comment (ColdSlav) [inline TGC/events/Germany VS Hungary Events.txt]: Possible encoding error, please take a look at it.

File: TGC/localisation/00_events_1000000000.csv [hunk 2]
```diff
@@ -97,7 +97,7 @@ EVTOPTA2000000010;NEVER!;;;;;;;;;;;;;x
 EVTOPTB2000000010;Those regions are barely hungarian anyways;;;;;;;;;;;;;x
 EVTNAME2000000011;Germany demands we restore the Habsburg King to power;;;;;;;;;;;;;x
 EVTDESC2000000011;Germany has send us an Ultimatum to restore the Habsburg Monarch to absolute Power and submit to them carving up our country.;;;;;;;;;;;;;x
-EVTOPTA2000000011;Tell them to go fuck themselfs!;;;;;;;;;;;;;x
+EVTOPTA2000000011;Tell them to go fuck themselves!;;;;;;;;;;;;;x
```
Comment (ColdSlav) [inline TGC/localisation/00_events_1000000000.csv]: Language.
```suggestion
EVTOPTA2000000011;Tell them to go to hell!;;;;;;;;;;;;;x
```

File: TGC/localisation/00_events_1000000000.csv [hunk 3]
```diff
@@ -259,19 +259,19 @@ EVTOPTA2000000058;report this on the discord if you do;;;;;;;;;;;;;x
...
-EVTNAME2000000061;An embarressing conference;;;;;;;;;;;;;x
-EVTDESC2000000061;There we stood with out hands completely empty. The Hungarians brought maps over maps that showed the region having more Hungarian inhabitants than germans. With no Romanian representative to argue against it, the Hungarians claimed the Romanians are happy with the current arrangement and mostly see themselfs as Hungarian anyways. In the end it was not them to decide however. They relayed the outcome of the conference to the Hungarian Government which statet that we.....;;;;;;;;;;;;;x
+EVTNAME2000000061;An embarrassing conference;;;;;;;;;;;;;x
+EVTDESC2000000061;There we stood with out hands completely empty. The Hungarians brought maps over maps that showed the region having more Hungarian inhabitants than germans. With no Romanian representative to argue against it, the Hungarians claimed the Romanians are happy with the current arrangement and mostly see themselves as Hungarian anyways. In the end it was not them to decide however. They relayed the outcome of the conference to the Hungarian Government which statet that we.....;;;;;;;;;;;;;x
```
Comment (ColdSlav) [inline TGC/localisation/00_events_1000000000.csv]:
```suggestion
EVTDESC2000000061;There we stood with our hands completely empty. The Hungarians brought maps over maps that showed the region as having more Hungarian inhabitants than Germans. With no Romanian representative to argue against it, the Hungarians claimed that the Romanians are happy with the current arrangement and mostly see themselves as Hungarian anyways. In the end, it was not for them to decide however. They relayed the outcome of the conference to the Hungarian Government which statet that we...;;;;;;;;;;;;;x
```

File: TGC/localisation/00_events_1000000000.csv [hunk 4]
```diff
@@ -259,19 +259,19 @@ EVTOPTA2000000058;report this on the discord if you do;;;;;;;;;;;;;x
...
 EVTOPTB2000000062;Agreed all of Transylvania be given autonomy as a Freistaat;;;;;;;;;;;;;x
 EVTNAME2000000063;Yelling and accusations at the conference;;;;;;;;;;;;;x
 EVTDESC2000000063;Our delegations had not even sat down properly when the first provokations were exchanged between the Romanians and Hungarians. Our Year long desinfo campaign seems to have worked as almost half conference consisted of Romanians accusing Hungary of plotting their forced assimilation or even worse. Centurys old incidents between the two were brought up and loudly debated whilst our delegation rarely got to speak. When we got the word our Diplomats played the calm and mediating voice. Every diplomat in the room understood that the two peoples could no longer share a country. In the end it was not theirs to decide however. They relayed the outcome of the conference to the Hungarian Government which stated that we.....;;;;;;;;;;;;;x
-EVTNAME2000000064;The Germans embaress themselfs;;;;;;;;;;;;;x
-EVTNAME2000000066;The Transylvania confernece;;;;;;;;;;;;;x
+EVTNAME2000000064;The Germans embarrass themselves;;;;;;;;;;;;;x
```
Comment (ColdSlav) [inline TGC/localisation/00_events_1000000000.csv]:
```suggestion
EVTNAME2000000064;The Germans Embarrass Themselves;;;;;;;;;;;;;x
```

File: TGC/localisation/00_events_1000000000.csv [hunk 5]
```diff
@@ -259,19 +259,19 @@ EVTOPTA2000000058;report this on the discord if you do;;;;;;;;;;;;;x
...
+EVTNAME2000000066;The Transylvania conference;;;;;;;;;;;;;x
```
Comment (ColdSlav) [inline TGC/localisation/00_events_1000000000.csv]:
```suggestion
EVTNAME2000000066;The Transylvania Conference;;;;;;;;;;;;;x
```

File: TGC/localisation/00_events_1000000000.csv [hunk 6]
```diff
@@ -259,19 +259,19 @@ EVTOPTA2000000058;report this on the discord if you do;;;;;;;;;;;;;x
...
 EVTNAME2000000068;Romanian Fury;;;;;;;;;;;;;x
-EVTDESC2000000064;The Germans forced another conference to again chip away at our sovereignty. However this time they went in completely unprepared and made absolute fools of themselfs. Despite their poor performance at at the table they still demanded that we give Transylvania proper Autonomy as a Freistaat. Their demands are probably backed by their military...;;;;;;;;;;;;;x
-EVTDESC2000000065;The Germans forced another conference to again chip away at our sovereignty. However this time they went in completely unprepared and made absolute fools of themselfs. Despite their poor performance at at the table they still demanded that we give all of Transylvania Autonomy as a Freistaat. Their demands are probably backed by their military...;;;;;;;;;;;;;x
+EVTDESC2000000064;The Germans forced another conference to again chip away at our sovereignty. However this time they went in completely unprepared and made absolute fools of themselves. Despite their poor performance at at the table they still demanded that we give Transylvania proper Autonomy as a Freistaat. Their demands are probably backed by their military...;;;;;;;;;;;;;x
```
Comment (ColdSlav) [inline TGC/localisation/00_events_1000000000.csv]:
```suggestion
EVTDESC2000000064;The Germans forced another conference to once again chip away at our sovereignty. This time however, they went in completely unprepared and made absolute fools of themselves. In spite of their poor performance at the table, they still demanded that we give Transylvania proper autonomy as a Freistaat. Their demands are probably backed by their military...;;;;;;;;;;;;;x
```

File: TGC/localisation/00_events_1000000000.csv [hunk 7]
```diff
@@ -259,19 +259,19 @@ EVTOPTA2000000058;report this on the discord if you do;;;;;;;;;;;;;x
...
+EVTDESC2000000065;The Germans forced another conference to again chip away at our sovereignty. However this time they went in completely unprepared and made absolute fools of themselves. Despite their poor performance at at the table they still demanded that we give all of Transylvania Autonomy as a Freistaat. Their demands are probably backed by their military...;;;;;;;;;;;;;x
```
Comment (ColdSlav) [inline TGC/localisation/00_events_1000000000.csv]:
```suggestion
EVTDESC2000000065;The Germans forced another conference to once again chip away at our sovereignty. This time however, they went in completely unprepared and made absolute fools of themselves. In spite of their poor performance at the table, they still demanded that we give all of Transylvania autonomy as a Freistaat. Their demands are probably backed by their military...;;;;;;;;;;;;;x
```

File: TGC/localisation/00_events_1000000000.csv [hunk 8]
```diff
@@ -280,10 +280,10 @@ EVTOPTA2000000064;We dont have a choice;;;;;;;;;;;;;x
...
+EVTDESC2000000070;After an embarrassing round of talks with the Hungarians, we have decided to not further pursue Transylvania. Our initial optimism about how easy the conference would be for us was clearly misplaced. We should have prepared better.;;;;;;;;;;;;;x
 EVTOPTA2000000070;What a shitshow;;;;;;;;;;;;;x
 EVTNAME2000000071;Transylvania defended;;;;;;;;;;;;;x
-EVTDESC2000000071;The Germans have dragged us to another round of negotiations in an attempt to chop away at our country. However this time they made complete fools of themselfs claiming Transylvania on basis of a small minority of German settlers. Even they recognised the futility of their efforts this time and agreed to recognise the region as ours.;;;;;;;;;;;;;x
+EVTDESC2000000071;The Germans have dragged us to another round of negotiations in an attempt to chop away at our country. However this time they made complete fools of themselves claiming Transylvania on basis of a small minority of German settlers. Even they recognised the futility of their efforts this time and agreed to recognise the region as ours.;;;;;;;;;;;;;x
```
Comment (ColdSlav) [inline TGC/localisation/00_events_1000000000.csv]:
```suggestion
EVTDESC2000000071;The Germans have dragged us to another round of negotiations in an attempt to chop away at our country. However this time they made complete fools of themselves claiming Transylvania on basis of a small minority of German settlers. Even this group of settlers recognised the futility of their efforts this time and agreed to recognise the region as ours.;;;;;;;;;;;;;x
```

**Review by ColdSlav — CHANGES_REQUESTED** [review]
Overall a pretty good work, there are some other errors that need addressing before I approve this merge though.

File: TGC/events/Germany VS Hungary Events.txt (hunk 1 above)
Comment (joelAVS) [reply TGC/events/Germany VS Hungary Events.txt]: Just a missing character in the comment, nothing to worry about. But I fixed it anyway

**Review by ColdSlav — APPROVED** [review]

**Comment by ColdSlav** [comment]
Looks alright

File: TGC/localisation/00_events_1000000000.csv [hunk 9]
```diff
@@ -1,14 +1,14 @@
...
 EVTNAME1904191418;$FROMCOUNTRY$ Has Refused;;;;;;;;;;;;;x
 EVTOPTA1904191418;We will accept their decision.;;;;;;;;;;;;;x
 EVTOPTB1904191418;Willingly or not, they will join.;;;;;;;;;;;;;x
 EVTDESC1904191419;Wyke-Aycinena Convention of $YEAR$;;;;;;;;;;;;;x
-EVTNAME1904191419;Our foreign minister Pedro de Aycinena y Piñol has negotiated a deal with British Consul Charles Lennox Wyke. In exchange for our acknowledgement that the territory between the Belize and Sarstoon Rivers is a part of the British settlement, and not $COUNTRY$, the British will allow us to purchase needed weapons and to build a road between Ciudad de Guatemala and Belize Town to improve communication with our Atlantic coast.;;;;;;;;;;;;;x
+EVTNAME1904191419;Our foreign minister Pedro de Aycinena y PiÃ±ol has negotiated a deal with British Consul Charles Lennox Wyke. In exchange for our acknowledgement that the territory between the Belize and Sarstoon Rivers is a part of the British settlement, and not $COUNTRY$, the British will allow us to purchase needed weapons and to build a road between Ciudad de Guatemala and Belize Town to improve communication with our Atlantic coast.;;;;;;;;;;;;;x
```
Comment (ColdSlav) [inline TGC/localisation/00_events_1000000000.csv]: I hope this is a GitHub issue and not an encoding error

Comment (ColdSlav) [reply TGC/localisation/00_events_1000000000.csv]: (It's an encoding error)

Comment (ColdSlav) [reply TGC/localisation/00_events_1000000000.csv]:
```suggestion
EVTNAME1904191419;Our foreign minister Pedro de Aycinena y Piñol has negotiated a deal with British Consul Charles Lennox Wyke. In exchange for our acknowledgement that the territory between the Belize and Sarstoon Rivers is a part of the British settlement, and not $COUNTRY$, the British will allow us to purchase needed weapons and to build a road between Ciudad de Guatemala and Belize Town to improve communication with our Atlantic coast.;;;;;;;;;;;;;x
```

Comment (ColdSlav) [reply TGC/localisation/00_events_1000000000.csv]: There we go

**Pull request merged**